deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -13,14 +13,14 @@ data:
     @include common.conf
     @include metrics.conf
     @include logs.conf
-  buffer.output.conf: |-
+  buffer.output.conf: |
```

Comment:
hmm not sure where this change is coming from

Reply:
maybe the addition of the newline character in `buffer.output.conf`? Looks like it didn't have a trailing newline before

Reply:
oh interesting, let me try removing it

Reply:
that fixed it 👀 thanks Ryan